projects/gen-ai.md

```diff
@@ -0,0 +1,157 @@
+# Generative AI Observability
+
+>[!IMPORTANT] This project extends LLM Semantic Conventions project
+
+## Description
+
+Generative AI observability is evolving fast. The industry is still learning how to use GenAI products which creates new opportunities and challenges in the observability space. GenAI observability is focused on development time and verbose telemetry. Non-deterministic nature of GenAI creates the need to measure the quality and safety of responses which become part of observability.
+
+There are multiple GenAI-focused observability projects out there such as [Arize Phoenix](https://docs.arize.com/phoenix), [Langfuse](https://langfuse.com/), [LangSmith](https://www.langchain.com/langsmith), [Langtrace](https://langtrace.ai/), [OpenLit](https://github.com/openlit/openlit), [Traceloop](https://www.traceloop.com/docs/introduction) and many others.
+
+Some of these projects are based on OpenTelemetry, others are "inspired by" it, but don't have out-of-the-box integration with OTel.
+
+The scope of GenAI observability includes instrumentations of various GenAI clients, frameworks, response evaluations, as well as model (server-side) instrumentations.
```

Comment:
this represents what some of the vendors do now, and some work that's not happened yet (server-side for example is not well explored by most involved). Is this the new scope or is this a motivation example, and not necessarily a definition of the resulting scope?

Reply:
This sentence talks about overall GenAI observability and covers areas that were brought up by the community in the past several months. LLM server metrics are already in semantic conventions. Based on my anecdotal experience, service-side RAG (model querying database) is another popular server-side area.  

Based on my experience working in messaging SIG, some providers interested in instrumenting server side come to the SIG to learn more about  tracing and metrics and what they can do on the server. 

So I do think that some of the service-side work is inevitable.
At the same time, there is no specific server-side work mentioned in the deliverables.